components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -502,6 +502,18 @@ curl -sSH "api-token: $TOKEN" -X DELETE \
 https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/projects/{project-id}/rules/{rule-id}?pretty
 ```
 
+#### Applying Rules
```

Comment:
did you mean for this to be an h4 under deletein a project rule?
think it should prolly be an h3...

Reply:
I was opting to make it its own section, since it's sort of wed to both projects and project rules. I can do an h3 though if you'd prefer.

Reply:
but its not a sub of the delete rule content right?

Reply:
you could also do h2 if you dont want it to be a sub of project rules

Reply:
oh lol wrong direction